galaxy.yml

```diff
@@ -1,6 +1,6 @@
 namespace: community
 name: aws
-version: 2.3.0
+version: 2.4.0
 readme: README.md
 authors:
   - Ansible (https://github.com/ansible)
```

Comment:
Do we need to update dependencies with amazon.aws 2.2.0? Or amazon.aws >= 2.1.0 is fine?

Reply:
I don't know of any new dependencies that were introduced. In theory, since 3.x.x is released, the 2.x.x releases should receive only bugfixes and docfixed.  
But I'm fine to increase also the minimum `amazon.aws` requirements, because it introduces only bugixes :)